### ISSUE TITLE
Toast Messages Display Weird

### ISSUE THREAD

**Issue by OldBuffGuy** [tachiyomiorg/tachiyomi#9964]
### Steps to reproduce

1. Tap More
2. Tap Settings
3. Tap Advanced
4. Tap Clear chapter cache

### Expected behavior

Should see Material You toast message with the Tachiyomi app icon, like "<app-icon> cache cleared. _ files have been deleted"

### Actual behavior

Toast message displays in rectangle, stating "Cache cleared. 0 files have been deleted"

### Tachiyomi version

0.14.6

### Android version

Android 13

### Device

Google Pixel 6 Pro

### Other details

Happens with all toast messages, not just the clearing cache one. The same result occurs in the latest Preview build as well.

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a short but informative title.
- [X] If this is an issue with an extension, I should be opening an issue in the [extensions repository](https://github.com/tachiyomiorg/tachiyomi-extensions/issues/new/choose).
- [X] I have gone through the [FAQ](https://tachiyomi.org/docs/faq/general) and [troubleshooting guide](https://tachiyomi.org/docs/guides/troubleshooting/).
- [X] I have updated the app to version **[0.14.6](https://github.com/tachiyomiorg/tachiyomi/releases/latest)**.
- [X] I have updated all installed extensions.
- [X] I will fill out all of the requested information in this form.

**Comment by OldBuffGuy** [comment]
![Screenshot_20230926-211955](https://github.com/tachiyomiorg/tachiyomi/assets/1783037/ffdb6f10-97a8-4308-b5ed-be8b8e7a1ebc)
![Screenshot_20230926-211738](https://github.com/tachiyomiorg/tachiyomi/assets/1783037/d34f5a9d-d3fb-42a0-a196-cce1cf6736d6)

**Comment by oyehanif** [comment]
I would like to work on this issue

**Comment by arkon** [comment]
Go ahead, but I suspect this is related to the app's target SDK which would be difficult to actually address without larger changes. Would need to confirm in a separate proof of concept or something.

**Comment by Maan-Sharma** [comment]
anyone work on this?

**Comment by sahil19solanki** [comment]
Hello, I want to work on this issue , Can I ?

**Comment by OldBuffGuy** [comment]
I don't know if you all are waiting on my approval... If you are, please go
ahead! I'm happy to help in any way I can!

**Comment by OldBuffGuy** [comment]
I have updated to Android 14 and the issue still persists.

**Comment by OldBuffGuy** [comment]
I have updated to 0.14.7 and the issue still persists.

**Comment by adersh1996** [comment]
Hello, I want to work on this issue , Can I ?

**Comment by OldBuffGuy** [comment]
You have my permission. Others have asked already too.

**Comment by adersh1996** [comment]
Thank you, I have created a pull request for the same

**Comment by arkon** [comment]
As noted in https://github.com/tachiyomiorg/tachiyomi/pull/10082#issuecomment-1783908302:

[My theory about the target SDK](https://github.com/tachiyomiorg/tachiyomi/issues/9964#issuecomment-1741773846) seems to be correct.

However since storage permissions changed in newer SDK versions, we need to first:

1. Consolidate storage locations (downloads, local source, automatic backups; and any necessary migrations or guides for existing users)
2. Switch to scoped storage permissions
3. Finally bump the target SDK, assuming there's no other SDK increase related changes to be done (there probably is)

**Comment by OldBuffGuy** [comment]
Sounds good, thanks Arkon!
I look forward to the change!
